src/GT.UI/Program.cs

```diff
@@ -40,21 +43,23 @@
 	.AddEntityFrameworkStores<GTIdentityContext>();
 
 // Add services to the container.
-builder.Services.AddControllersWithViews();
+builder.Services
+	.AddControllersWithViews()
+	.AddNewtonsoftJson(o =>
+	{
+		o.SerializerSettings.ReferenceLoopHandling = ReferenceLoopHandling.Ignore;
+	});
 
 // Add DAL repositories
 builder.Services
-	.AddScoped(typeof(IGTGenericRepository<Location>), typeof(GTGenericRepository<Location>))
-	.AddScoped(typeof(IGTGenericRepository<Company>), typeof(GTGenericRepository<Company>))
-	.AddScoped(typeof(IGTGenericRepository<Listing>), typeof(GTGenericRepository<Listing>))
-	.AddScoped(typeof(IGTGenericRepository<ListingInquiry>), typeof(GTGenericRepository<ListingInquiry>));
+	.AddTransient(typeof(IGTGenericRepository<>), typeof(GTGenericRepository<>))
+	.AddTransient<IGTIdentityRepository, GTIdentityRepository>();
 
 // Add BLL services
 builder.Services
-	.AddScoped<IGTLocationService, GTLocationService>()
-	.AddScoped<IGTCompanyService, GTCompanyService>()
-	.AddScoped<IGTListingService, GTListingService>()
-	.AddScoped<IGTListingInquiryService, GTListingInquiryService>();
+	.AddTransient<IGTListingService, GTListingService>();
+
+builder.Services.AddAutoMapper(Assembly.GetExecutingAssembly());
```

Comment:
Duplication on AddAutoMapper? (see row 64)

Reply:
Fix in commit.